uportal-war/src/main/java/org/jasig/portal/groups/pags/dao/jpa/JpaPersonAttributesGroupDefinitionDao.java

```diff
@@ -86,64 +117,37 @@ public void deletePersonAttributesGroupDefinition(IPersonAttributesGroupDefiniti
         entityManager.remove(persistentDefinition);
     }
 
-    @PortalTransactional
```

Comment:
Do we need @PortalTransactionalReadOnly here?  (I'm pretty sure we don't need @PortalTransactional).